Seeing AttributeError: 'SubRequest' object has no attribute 'path' since 1.6

Hello,

We're seeing the failure

```
  @pytest.fixture(scope="module")
    def original_datadir(request):
>       return request.path.parent / request.path.stem
E       AttributeError: 'SubRequest' object has no attribute 'path'

../../../venv3/lib64/python3.11/site-packages/pytest_datadir/plugin.py:45: AttributeError
```

since release of 1.6 an hour ago.

is there a version requirement for request? where should it be from?

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -33,6 +33,19 @@ jobs:
       matrix:
         python: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
         os: [ubuntu-latest, windows-latest, macos-latest]
+        include:
+          - python: "3.8"
+            tox_env: "py38"
+          - python: "3.9"
+            tox_env: "py39"
+          - python: "3.10"
+            tox_env: "py310"
+          - python: "3.11"
+            tox_env: "py311"
+          - python: "3.12"
+            tox_env: "py312"
+          - python: "3.13"
+            tox_env: "py313"
 
     steps:
     - uses: actions/checkout@v4
@@ -56,4 +69,4 @@ jobs:
     - name: Test
       shell: bash
       run: |
-        tox run -e py --installpkg `find dist/*.tar.gz`
+        tox run -e ${{ matrix.tox_env }} --installpkg `find dist/*.tar.gz`

--- a/.gitignore
+++ b/.gitignore
@@ -60,4 +60,4 @@ target/
 # IDEs
 .idea/
 
-pytest_datadir/_version.py
+/src/pytest_datadir/_version.py

--- a/CHANGELOG.rst
+++ b/CHANGELOG.rst
@@ -1,8 +1,17 @@
 pytest-datadir
 ==============
 
+1.6.1
+-----
+
+*2025-02-07*
+
+- pytest 7.0+ is now required.
+
 1.6.0
 -----
+
+**Note**: this release has been yanked from PyPI due to `#89 <https://github.com/gabrielcnr/pytest-datadir/issues/89>`__.
 
 *2025-02-07*
 

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -27,7 +27,8 @@ classifiers = [
 dynamic = ["version"]
 requires-python = ">=3.8"
 dependencies = [
-    "pytest>=5.0",
+    # Update tox.ini if the minimum required version changes.
+    "pytest>=7.0",
 ]
 
 [project.entry-points.pytest11]

--- a/tox.ini
+++ b/tox.ini
@@ -2,5 +2,8 @@
 envlist = py{38,39,310,311,312}
 
 [testenv]
+deps =
+    # Test with minimum required pytest version.
+    py38: pytest==7.0.0
 commands=
     pytest --color=yes {posargs:tests}

